research/src/pages/tabs.research.parts.mdx

```diff
@@ -0,0 +1,439 @@
+
+
+
+## Disabled Tabs
+A few systems allow the "disabling" of individual tab labels.  While this is not extremely common, it is not entirely rare either. Some particularly interesting things to note here are:
+
+* The accessibility characteristics of this are [not well established or agreed upon](https://github.com/w3c/aria-practices/issues/1237)
+* There is a slight conflict here in that many design guidelines recommendations for when to use a Tab Set (whatever it is called) describe a quality that tabs should not be statefully related, and most of the use cases for this quality seem to have that as a prerequisite (wizard-like sequential forms scenarios, for example).
+
+
+<figure>
+  <img src="https://lh4.googleusercontent.com/5TLmXxzF3AvHBr-ii7tkUDAoc3-XpBDe-KYZ4QdIzKUSNolQO0WY-S5j8cAB3huPoM-a43WHqtGahqUgDOdsewpb435LAJuDoiIopxao5McvDVogt-pcvuBGDG-mt6JIhRN-JqVK" />
+  <figcaption>Ant design supports disabled tab labels</figcaption>
+</figure>
+
+
+<figure>
+  <img src="https://lh6.googleusercontent.com/ro6YMMbueRFJGaDzcHx7Jxuu5ya6zJW53i63gcpnpB5E_wHUdoFhuV01NbCjtxzellFRSQhwNFhNzfsX7ithKTidAuvFRcWl_Fhhoek7BQLZyMq6FLrBmnaZOiUtF5KCBaBv3s3P" />
+  <figcaption>Carbon design supports disabled tab labels</figcaption>
+</figure>
+
+
+<figure>
+  <img src="https://lh3.googleusercontent.com/r_wnzafQTxPyTYTnwrfeQ1VK2m77zIUtz5CtWcW-1X2OF1q0kUuWs1PZ-iUhjHfjcp-hjSxF1GR0HlviYODjC6INwblpsPLhHLPxUytMclmCRaGBeCSKD4NLBqmCle7arnTRlaJc" />
+  <figcaption>Shoelace supports disabled tab labels</figcaption>
+</figure>
+
+
+## Dismiss / Close / Delete / Remove
+Some Tab Sets allow programatic and sometimes user-triggered items that are 
+"dismissible" or, as referred to in ARIA practices as "deletable" or, sometimes "closable" or "removable".  
+
+Note that while this optional feature is mentioned in ARIA Practices and keyboard affordances are well-specified, there seem to be disagreements in implementations about whether a semantic/visual affordance belongs in the sequential focus order.
+
+Note that if tabsets are 'dismissable' they can also be 'entirely dismissable' (the entire control disappears) or they have to deal with what happens when only one tab remains.  It seems the choice to support this, not support it, or how to deal with whether they are individually, as a whole, or only as long as more than one remains varies quite a bit, though realistically without 2 (that are not disabled) it gets more difficult to really call them "tabs", thus many design systems at least recommend, if not require  minimum of two.  In many windowing toolkits, "dismissable" cases are dealt with extensions to a common interface and not called 'tabs', or 'tabs' has a narrower definition in one direction or the other.
+ 
+ This might be thought of to go hand in hand with the ability to add a tab, and while not uncommon in programmatic APIs, it is considerably more rare from a user interface point of view and there aren't plentiful ideas in practice from libraries to draw from.
```

Comment:
Just an FYI, we have this all throughout Salesforce UI.

Reply:
sorry it's hard to tell what the 'this' is here - can you be more specific?